simulator/simulator.go

```diff
@@ -0,0 +1,197 @@
+package simulator
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/env"
+	"github.com/bitrise-io/go-utils/errorutil"
+	"github.com/bitrise-io/go-utils/log"
+	sim "github.com/bitrise-io/go-xcode/simulator"
+)
+
+// InfoModel ...
+type InfoModel sim.InfoModel
+
+// Simulator ...
+type Simulator interface {
```

Comment:
I'd argue this interface doesn't represent a simulator, it's something that manages simulators and such. Should we consider a different name?

Reply:
what about `simulator.Manager` ?

Reply:
Sounds good to me (or `SimulatorManager`)!

Reply:
Some Go linter throws an error for `simulator.SimulatorManager` (`Name starts with the package name`).
Renamed to `simulator.Manager` in https://github.com/bitrise-steplib/steps-xcode-test/pull/181/commits/bfdf03e5fe90d5d138b63de39b0b0a5052d9d929